frontend/assets/markdown/privacy-policy.md

```diff
@@ -0,0 +1,9 @@
+**PLEASE READ CAREFULLY**: This IBM Quantum End User Agreement is between you (“You”),  and IBM, and consists of (a) the terms of this IBM Quantum End User Agreement, (b) the IBM Privacy Policies (defined below), and (c) the authorizations and related instructions indicated on Your IBM Quantum account page listing the specific IBM Quantum resources You are permitted to use (referred to herein as “Authorization Instructions”) (collectively, this “Agreement”).
+ 
+By accepting this Agreement on the IBM Quantum webpages, You (i) accept the terms of this Agreement, as may be amended from time to time as provided below and subject to the “Applicability” terms in Article 1.1 below, (ii) understand that You are entering a binding legal agreement, and (iii) are at least 18 years old.
+ 
+If You are under 14, You may not access IBM Quantum.
+ 
+If You are between 14 and 17 years old, then Your parent or legal guardian must accept this Agreement for You. By doing so, Your parent or legal guardian represents and warrants s/he has the legal right to do so, understands that this is a binding legal agreement, and accepts these terms on Your behalf.
+
+This Agreement replaces all prior versions of the “IBM Q End User Agreement”, “IBM Q End User License Agreement” and the “IBM Q Experience End User License Agreement” and any previously linked “Terms of Use” or “Acceptable Use Policy”.
```

Comment:
I'm afraid this is just part of the privacy police text.

You can see the full text here: https://quantum-computing.ibm.com/terms